### PR TITLE
chore: small changes to dropdown & main sidebar

### PR DESCRIPTION
**Pull request by ciaranha**
## What does this PR do?

Small changes to filter dropdown. Improved border colors, font-weights, focus state, icons size, spacing around dividers. Other small changes to sidebar. Making items a little more compact.

https://github.com/calcom/cal.com/assets/4536123/cf3588dc-57f8-4080-b83c-ef0d9524a9ca

## Type of change

- UI fix (non-breaking change which fixes an issue)

## How should this be tested?

- [ ] Go to Routing forms and check out the filter dropdown.
- [ ] Test the User dropdown in the main sidebar

### CODE REVIEW

**Review by sean-brydon — APPROVED** [review]
LGTM - Glad us setting up your env didnt take hours <3

File: packages/features/shell/Shell.tsx [hunk 1]
```diff
@@ -340,24 +340,24 @@ function UserDropdown({ small }: UserDropdownProps) {
...
           )}>
           <span
             className={classNames(
-              small ? "h-4 w-4" : "h-6 w-6 ltr:mr-2 rtl:ml-2",
+              small ? "h-4 w-4" : "h-5 w-5 ltr:mr-2 rtl:ml-2",
               "relative flex-shrink-0 rounded-full bg-gray-300"
             )}>
             <Avatar
-              size={small ? "xs" : "sm"}
+              size={small ? "xs" : "xsm"}
```
Comment (sean-brydon) [inline packages/features/shell/Shell.tsx:351]: Was this in intentional

**Review by sean-brydon — CHANGES_REQUESTED** [review]
Requesting changes while we figure something out

File: packages/features/shell/Shell.tsx (hunk 1 above)
Comment (Jaibles) [reply packages/features/shell/Shell.tsx:351]: Yea I want the avatar to be 20x20px - so if that's indeed what this does - then yes!

**Comment by PeerRich** [comment]
> Requesting changes while we figure something out

figured out? cause i tested everytrhing and it looks fine

**Review by PeerRich — DISMISSED** [review]

**Comment by sean-brydon** [comment]
> > Requesting changes while we figure something out
> 
> figured out? cause i tested everytrhing and it looks fine

Yeah all good - We spotted something with nav items but i dont think its coming from this PR - if it is i'll follow up tomorrow

**Review by sean-brydon — DISMISSED** [review]

**Review by PeerRich — DISMISSED** [review]

**Review by PeerRich — DISMISSED** [review]

File: packages/ui/components/empty-screen/EmptyScreen.tsx [hunk 2]
```diff
@@ -34,7 +34,7 @@ export function EmptyScreen({
       <div
         data-testid="empty-screen"
         className={classNames(
-          "min-h-80 flex w-full flex-col items-center justify-center rounded-md p-7 lg:p-20",
+          "min-h-80 flex w-full flex-col items-center justify-center rounded-lg p-7 lg:p-20 select-none",
```
Comment (PeerRich) [inline packages/ui/components/empty-screen/EmptyScreen.tsx:37]:
```suggestion
          "select-none min-h-80 flex w-full flex-col items-center justify-center rounded-lg p-7 lg:p-20",
```

File: packages/ui/components/popover/AnimatedPopover.tsx [hunk 3]
```diff
@@ -47,28 +47,28 @@ export const AnimatedPopover = ({
         <div
           ref={ref}
           className={classNames(
-            "hover:border-emphasis border-default text-default hover:text-emphasis mb-4 flex h-9 max-h-72 items-center justify-between whitespace-nowrap rounded-md border px-3 py-2 text-sm hover:cursor-pointer focus:border-neutral-300 focus:outline-none focus:ring-2 focus:ring-neutral-800 focus:ring-offset-1",
+            "hover:border-emphasis border-default text-default hover:text-emphasis radix-state-open:border-emphasis radix-state-open:outline-none radix-state-open:ring-2 radix-state-open:ring-emphasis mb-4 -mr-0.5 flex h-9 max-h-72 items-center justify-between whitespace-nowrap rounded-md border px-3 py-2 text-sm hover:cursor-pointer",
```
Comment (PeerRich) [inline packages/ui/components/popover/AnimatedPopover.tsx:50]:
```suggestion
            "hover:border-emphasis border-default text-default hover:text-emphasis radix-state-open:border-emphasis radix-state-open:outline-none radix-state-open:ring-2 radix-state-open:ring-emphasis -mr-0.5 mb-4 flex h-9 max-h-72 items-center justify-between whitespace-nowrap rounded-md border px-3 py-2 text-sm hover:cursor-pointer",
```

**Review by PeerRich — DISMISSED** [review]

File: packages/ui/components/empty-screen/EmptyScreen.tsx [hunk 4]
```diff
@@ -34,7 +34,7 @@ export function EmptyScreen({
       <div
         data-testid="empty-screen"
         className={classNames(
-          "min-h-80 flex w-full flex-col items-center justify-center rounded-md p-7 lg:p-20",
+          "select-none min-h-80 flex w-full flex-col items-center justify-center rounded-lg p-7 lg:p-20",
```
Comment (PeerRich) [inline packages/ui/components/empty-screen/EmptyScreen.tsx:37]:
```suggestion
          "min-h-80 flex w-full flex-col items-center justify-center rounded-lg p-7 lg:p-20 select-none",
```

**Review by PeerRich — DISMISSED** [review]

File: packages/ui/components/empty-screen/EmptyScreen.tsx (hunk 2 above)
Comment (PeerRich) [inline packages/ui/components/empty-screen/EmptyScreen.tsx:37]:
```suggestion
          "select-none flex-col items-center justify-center rounded-lg p-7 lg:p-20",
```

**Review by PeerRich — APPROVED** [review]

**Pull request merged**